### ISSUE TITLE
The upgrade firmware entity doen't dissapear after upgrade

### ISSUE THREAD

**Issue by astrandb** [StyraHem/hass#14]
I have upgraded a shelly from your suggested monster card. The upgrade went through without problems but the entity is still present. Is this behaviour intended?

**Comment by astrandb** [comment]
Well, a couple of hours later, with a number of restarts of HA not related to the shelly integration, the upgrade entity has disappeared.

**Comment by tefinger** [comment]
As I see in the current implementation it loads a switch platform for Shellies which have firmware update available. But it never unloads / removes the switch. Only after restarting HA the firmware update switches will be gone.

**Comment by hakana** [comment]
The button should be removed when you shitch it to on. I will fix so it will be removed when detecting right firmware but I didn't have tiome for that when I implemented it.

**Comment by tefinger** [comment]
BTW Since 0.0.11 I get these error message in the the log

```
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/homeassistant/helpers/entity_platform.py", line 357, in _async_add_entity
    raise HomeAssistantError(msg)
homeassistant.exceptions.HomeAssistantError: Entity id already exists: switch.shelly_shsw_1_554706_firmware_update```

**Comment by hakana** [comment]
Yes, that issue is solved. But I can't find a way to remove the switch entity after updating the firmware. 

I'm doing this but it has no effect on hass.

hass.add_job(update_switch.async_remove)

**Comment by hakana** [comment]
This should been fixed in latest release, 0.0.12
